### DISCUSSION TITLE
page will be empty after an hours

### DISCUSSION THREAD

**Discussion by alirezamehrabi** [vercel/next.js#56828] [Help]
### Summary

Hi, I have a MERN Stack website. in localhost it hasn't any problem but after build, it stop working after about one hour.

I deleted .next folder, node module and also server cache but nothing changed.

the page give 304 method in network tab and is compeletely empty

after build it works about an hour and stop working again

my website is "next": "^13.5.4" App router

my nodejs : v18.12.1

**Comment by mih0vil** [comment]
maybe this will help:
https://github.com/vercel/next.js/discussions/55893#discussioncomment-7131799
